app/_hub/kong-inc/grpc-web/index.md

```diff
@@ -45,13 +49,18 @@ params:
 
 ## Purpose
 
-A service that presents a gRPC API can be used by clients written in many languages, but the network specifications are oriented primarily to connections within a datacenter. [gRPC-Web] lets you expose this API to the Internet so that it can be consumed by browser-based JS apps.
+A service that presents a gRPC API can be used by clients written in many languages,
+but the network specifications are oriented primarily to connections within a datacenter.
```

Comment:
I'm not sure how to rephrase this, but it's hard to read as-is. The basic gist of this phrase is: you have a service with a gRPC API, you want to expose it to the internet, but the standard network specs don't provide that flexibility. 

Can we turn it into something more readable?

Reply:
All I did was wrap the original text. The scope of this PR is the new non-hub write-up, primarily. This sentence in conjunction with the next sentence makes sense to me.

Reply:
I'm going to edit the second sentence a bit...